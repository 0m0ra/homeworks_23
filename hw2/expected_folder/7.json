{"error": "Input file is empty."}