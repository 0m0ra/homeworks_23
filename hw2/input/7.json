{
    "user": {
        "region": "Saint-Petersburg",
        "registered": "201212-24",
        "last_login": "2023-10-01",
        "email": "user@yandex.ru",
        "age": 36
    },
    "user2": {
        "region": "Sochi",
        "registered": "2022-10-30",
        "last_login": "2022-11-21",
        "email": "user2@gmail.com",
        "age": 18
    },
    "user3": {
        "region": "Moscow",
        "registered": "2019-10-30",
        "last_login": "2022-11-21",
        "email": "@mail.ru",
        "age": 18
    }
}