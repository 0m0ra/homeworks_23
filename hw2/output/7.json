{
   "status": "ValueError",
   "message": "201212-24 in uncorrect format YYYY-MM-DD or time"
}