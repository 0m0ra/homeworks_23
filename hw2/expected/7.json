{
   "0-18": 0.0,
   "18-25": 0.0,
   "26-45": 50.0,
   "45-59": 0.0,
   "60+": 50.0,
   "2004": 50.0,
   "2005": 50.0
}